build/webpack.base.conf.js

```diff
@@ -21,7 +21,9 @@ module.exports = function baseConfig(env) {
       extensions: ['.ts', '.js', '.vue', '.json'],
       alias: {
         'vue$': 'vue/dist/vue.esm.js',
-        '@': utils.resolve('src')
+        '@': utils.resolve('src'),
```

Comment:
I've actually never tried it, should work I'll give it a shot to make sure

Reply:
@enochtsang I have made this change but I have to admit I'm not sure how I feel about it, being that you should be able to tell if something is a dir because it doesn't have an extension? I just feel like something like:
`path.resolve('utils/', 'test.ts')`
is less readable than 
`path.resolve('utils', 'test.ts')`

what are your thoughts?

Reply:
I disagree, I like `path.resolve('utils/', 'test.ts')` way better. @beaesguerra we'll take your opinion.